cysync/src/store/provider/syncProvider/index.tsx

```diff
@@ -998,46 +998,48 @@ export const SyncProvider: React.FC = ({ children }) => {
     await notifications.getLatest();
   };
 
-  const intervals: NodeJS.Timeout[] = [];
+  const emptyIntervals: NodeJS.Timeout[] = [];
```

Comment:
Don't store the reference to empty intervals

Reply:
Yeah cool